apps/glusterfs/allocator_simple_test.go

```diff
@@ -176,7 +176,7 @@ func TestSimpleAllocatorInitFromDbWithOfflineDevices(t *testing.T) {
 		// devices are added to the ring
 		node, err := cluster.NodeEntryFromClusterIndex(tx, 0)
 		tests.Assert(t, err == nil)
-		nodeId = node.Info.Id
+		//	nodeId := node.Info.Id
```

Comment:
@MohamedAshiqrh I need your help here. I don't remember doing this change, is it necessary?